CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts

```diff
@@ -0,0 +1,24 @@
+import { IssueOrPR } from "../lib/IssueOrPR";
+import { OctokitAction } from "../lib/OctokitAction";
+import { ProjectContent } from "../lib/ProjectContent";
+
+class CreateCardForIssueFromLabel extends OctokitAction {
+
+    protected async execute(): Promise<void> {
+        const labelPrefix = this.getInput("label-prefix");
+        const labelName = this.payload.label.name;
+        const project = await ProjectContent.FromProject(this, this.getInputNumber("project-id"));
+        if (labelName.startsWith(labelPrefix)) {
+            const columnName = labelName.substring(labelPrefix.length).trim();
+            const column = project.columnFromName(columnName);
+            if (column) {
+                await project.moveOrCreateCard(this.payload.issue as IssueOrPR, column.id);
+            }
+        } else {
+            this.log("Unexpected label name: " + labelName);
+        }
+    }
+}
+
+const action = new CreateCardForIssueFromLabel();
+action.run();
```

Comment:
if `action.run()` is async, you should use an [IIFE](https://anthonychu.ca/post/async-await-typescript-nodejs/):

```javascript
(async () => {
     await action.run();
})();
```

Reply:
I've made the `run` sync and used this IIFE trick inside. That keeps the boilerplate low.

Reply:
Reverted after discussion about GH actions behavior. 
>I can see it seems to be the rather proposed solution to use `action.run()` barely: https://github.com/actions/typescript-action